db/structure.sql

```diff
@@ -276,6 +276,7 @@ CREATE TABLE public.activity_logs (
 --
 
 CREATE SEQUENCE public.activity_logs_id_seq
+    AS integer
```

Comment:
What's causing this file to change?  Anything intentional?

Reply:
No... I'll revert it.